index.html

```diff
@@ -359,6 +192,50 @@ <h3 hidden>heading</h3>
                 <hr class="hr2">
                 </div>
         </footer>
- <script src="assets/js/main.js"></script>
+
+        <!--modals-->
+        <div id="project-modal" class="modal">
+            <div class="work-details-card-container">
+            <div class="header-container">
+                <div class="modal-header">
```

Comment:
- [ ] It would be nice to have the same fonts used in the Figma design on your website so it looks like the original design